apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts

```diff
@@ -49,7 +49,7 @@ export class CreateIntegration {
 
       await this.invalidateCache.invalidateQuery({
         key: buildIntegrationKey().invalidate({
-          _environmentId: command.environmentId,
+          _organizationId: command.organizationId,
```

Comment:
Here we specify the cache key and the level changed from environment to organization.